src/game/RandomizationOfShipsPositions.js

```diff
@@ -0,0 +1,85 @@
+import { SHIPS_LIST } from "./SHIPS_LIST";
+
+export const gameBoardArray = [];
+let direction;
+let Position;
+const GAME_BOARD_WIDTH = 10;
+
+const createGameBoardArray = () => {
+  for (let i = 0; i < GAME_BOARD_WIDTH; i++) {
+    gameBoardArray.push([]);
+    for (let j = 0; j < GAME_BOARD_WIDTH; j++) {
+      gameBoardArray[i].push(0);
+    }
+  }
+};
+createGameBoardArray();
+
+const directionRoll = () => {
+  Math.floor(Math.random() * 2) === 0
+    ? (direction = "vertical")
+    : (direction = "horizontal");
+};
+
+const makePosition = (k) => {
+  Position = [];
+  for (let i = 0; i < 2; i++) {
+    Position.push(Math.floor(Math.random() * GAME_BOARD_WIDTH));
+  }
+  if (direction === "horizontal") {
+    if (Number(`${Position[1]}`) + SHIPS_LIST[k].size > GAME_BOARD_WIDTH) {
+      Position[1] = Math.abs(Number(`${Position[1]}`) - SHIPS_LIST[k].size);
+    }
+  } else {
+    if (Number(`${Position[0]}`) + SHIPS_LIST[k].size > GAME_BOARD_WIDTH) {
+      Position[0] = Math.abs(Number(`${Position[0]}`) - SHIPS_LIST[k].size);
+    }
+  }
+
+  return { Position, direction };
+};
+
+const addShip = () => {
+  for (let k = 0; k < SHIPS_LIST.length; k++) {
+    directionRoll();
+    if (direction === "horizontal") {
+      horizontal(k);
+    } else {
+      vertical(k);
+    }
+  }
+};
+
+const vertical = (k) => {
```

Comment:
Function `vertical` doesn't give any information about what it does. Only context allows us to predict it.